docs/guides/pdf-non-ascii-languages.rst

```diff
@@ -1,5 +1,5 @@
-Build PDF format for non-ASCII languages
-========================================
+Sphinx PDFs for non-ASCII languages
```

Comment:
non-ASCII should probably be something else. 
```suggestion
Sphinx PDFs for languages other than English
```

Reply:
Technically it's non-latin languages. How about "Sphinx PDFs for Unicode Languages" or "Sphinx PDFs for Non-European Languages"?

Reply:
Yea, I didn't know the best way of putting it. I'm fine with Unicode, as it seems the most correct. 👍